scripts/deploy-helm.sh

```diff
@@ -171,11 +171,8 @@ API_TOKEN_SCOPES=('"logs.ingest"' '"metrics.ingest"' '"ReadConfig"' '"WriteConfi
 check_s3_url
 
 if [ -z "$GCP_PROJECT" ]; then
-  GCP_PROJECT=$(gcloud config get-value project 2>/dev/null)
-  if [ -z "$GCP_PROJECT" ]; then
-    err "Invalid GCP_PROJECT. Set correct gcpProjectId in values.yaml"
-    exit 1
-  fi
+  err "Invalid GCP_PROJECT. Set correct gcpProjectId in values.yaml"
```

Comment:
(minor) Kamil, sorry I didn't mention earlier, we have `check_if_parameter_is_empty` function in lib,sh and we could use it here.

Reply:
Ok. The question is do we have to use it or just keep the fix as is?

Reply:
I'd prefer to change it especially that message is about missing required parameter which is a bit more adequate than `Invalid GCP_PROJECT` and we will reuse our existing function.

Reply:
Updated.